Automattic/jetpack
# Dashboard: add card to highlight the blocks available in Jetpack

#### Changes proposed in this Pull Request:

* ﻿Let's promote that Jetpack includes blocks in the At A Glance section of the Jetpack dashboard.

<img width="707" alt="Screenshot 2022-03-16 at 18 14 28" src="https://user-images.githubusercontent.com/426388/158655623-42270b00-b93b-4822-bdeb-e0ccd19117b5.png">


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Dashboard and scroll down.
* You should see a card promoting Jetpack blocks.
* Click on the support link, and ensure the link works.
* Go to Plugins > Add New, and install / activate the Classic Editor plugin.
* Go back to Jetpack > Dashboard, the blocks card should be gone.
